### PR TITLE
common code cleanup part 9

### PR DESCRIPTION
**Pull request by commy2**
ref: #2012

### CODE REVIEW

File: addons/common/functions/fnc_dumpArray.sqf [hunk 1]
```diff
@@ -14,11 +25,14 @@ for "_i" from 0 to _depth do {
 _depth = _depth + 1;
 
 if (IS_ARRAY(_var)) then {
-    if ((count _var) > 0) then {
+    if (count _var > 0) then {
```
Comment (jokoho48) [inline addons/common/functions/fnc_dumpArray.sqf]: _var isEqualTo []

Comment (commy2) [inline addons/common/functions/fnc_dumpArray.sqf]: noted

**Comment by commy2** [comment]
Ready to merge.

**Comment by MikeMatrix** [comment]
:+1: for correct use of reference :D

**Comment by jokoho48** [comment]
wuhu finaly gz @commy2 :D

**Pull request merged**